DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/igmp/test_igmp_snooping.py

```diff
@@ -0,0 +1,482 @@
+import math
+import asyncio
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_mdb import BridgeMdb
+from dent_os_testbed.lib.bridge.bridge_link import BridgeLink
+from dent_os_testbed.lib.ip.ip_link import IpLink
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_setup_streams,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_clear_traffic_items,
+)
+
+pytestmark = [pytest.mark.suite_functional_igmp,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures('cleanup_bridges')]
+
+GENERAL_QUERY_IP = '224.0.0.1'
+IGMP_LEAVE_V2_IP = '224.0.0.2'
+DEVIATION_BPS = 1000
+
+
+async def get_bridge_mdb(device_name, options='-d -s -j'):
+    """
+    Get parsed bridge Mdb entries
+    Args:
+        device_name (str): Dent device host_name
+        options (str): Options to pass to bridge cmd
+    Returns:
+        Dictinary with parsed mdb and router entries
+    """
+
+    out = await BridgeMdb.show(
+        input_data=[{device_name: [
+            {'options': options}]}], parse_output=True)
+    err_msg = f'Verify bridge mdb show cmd was successful \n{out}'
+    assert out[0][device_name]['rc'] == 0, err_msg
+
+    router_entires = out[0][device_name]['parsed_output'][0]['router']
+    mdb_entires = out[0][device_name]['parsed_output'][0]['mdb']
+    return mdb_entires, router_entires
```

Comment:
Why not return out[0][device_name]['parsed_output'][0]['router'], out[0][device_name]['parsed_output'][0]['mdb']